second 0x01 for python3
